Kmeans_python/silhouette.py

```diff
@@ -25,4 +34,18 @@ def silhouette(X, k_array):
     
     """
 
-    pass
+    scores = []
+    for i in range(len(k_array)):
+        centers, labels = fit(k_array[i])
+        score = silhouette_score(X, labels)
```

Comment:
I was under the impression we'd compute these manually so that our package is completely independent of sklearn. Understand the time constraint, and its fine to leave it as is if the implementation is not straightforward